assets/styles/components/footer.scss

```diff
@@ -15,34 +15,33 @@
 }
 
 footer {
-  border-top: 2px #CCC solid;
+  border-top: 2px #ccc solid;
```

Comment:
can we make this a variable?

Reply:
Hmm, comparing the styles to https://ho-design-system-test.herokuapp.com/components it looks like it should actually be `#aaa` anyway - I'll fix it and name it `dark-grey` or something.